pkg/controller/infrastructure/actuator_reconcile_test.go

```diff
@@ -102,111 +103,6 @@ var _ = Describe("Infrastructure Reconcile", func() {
 			},
 		}
 
-		Eventually(Object(natGateway)).Should(SatisfyAll(
```

Comment:
Why has this code been removed?

Reply:
I thought testcase was duplicated, but I was wrong. Reverting changes.